app/components/CounterButton/CounterButton.Styled.jsx

```diff
@@ -9,13 +9,22 @@ const handleColorType = (props) => {
   return '#31425a';
 };
 
+const setCursorType = (props) => {
+  if (props.processingFormSubmission) {
+    return 'not-allowed;';
+  }
+  else {
+    return 'pointer';
+  }
+};
+
 export const ContainerCounterButton = styled.button`
   align-items: center;
   background-color: transparent;
   border: none;
   border-radius: 3px;
   color: ${props => handleColorType(props)};
-  cursor: pointer;
+  cursor: ${props => setCursorType(props)};
```

Comment:
This calling.

Reply:
Exactly, the semicolon is already there, no need for it in the setCursorType function, thank you!